src/map/script.cpp

```diff
@@ -21572,6 +21572,44 @@ BUILDIN_FUNC(instance_live_info)
 	}
 	return SCRIPT_CMD_SUCCESS;
 }
+/*==========================================
+ * set '.@instance_list' to a list of the live instance ids for the map with the mode.
+ * return the array size of '.@instance_list'
+ * .@size = instance_list(IM_NONE, "prontera");
+ *------------------------------------------*/
+BUILDIN_FUNC(instance_list)
+{
+	if (!instance_start) {
+		script_pushint(st, 0);
+		return SCRIPT_CMD_SUCCESS;
```

Comment:
I dont know if this scenario can even happen, but if it does then please return SCRIPT_CMD_FAILURE and tell the user that it happened.

Reply:
I intended for this to just be a short cut if no one in the server started any instance it would return 0 as would be no instance's map in the server anyway.
it's would be handled in the for loop as there would be no loop if it's 0. I will remove the check.